### DISCUSSION TITLE
Too much gap between navigation groups

### DISCUSSION THREAD

**Discussion by khanakia** [filamentphp/filament#9610] [Help]
There is too much gap between Navigation Group? I also see the demo.filamentphp.com has `gap-y-1`  so how can i achieve the same ?
```
-mx-2 flex flex-col gap-y-7
```

![Screenshot 2023-11-13 at 1 59 30 PM](https://github.com/filamentphp/filament/assets/2174170/c0785314-56f5-45c6-9ad1-4c1a6ee0a89f)

![Screenshot 2023-11-13 at 1 59 07 PM](https://github.com/filamentphp/filament/assets/2174170/3b081848-27fd-485d-af1f-2a5c6a79fc61)

**Comment by sinnbeck** [comment]
Add a override in your theme.

```css
.fi-sidebar-nav {
    @apply !gap-y-1;
}

**Reply by khanakia** [reply]
Can you point out which file should i add this as i am just using default Filament theme.  

Also i wonder how they achieved in demo https://github.com/filamentphp/demo as i do not see the above code anywhere in their codebase you sent me

**Reply by leandrocfe** [reply]
it should be gap-y-1.. what is the filament version?

**Reply by leandrocfe** [reply]
check this file `vendor/filament/filament/resources/views/components/sidebar/index.blade.php`
```php
        @if (filament()->hasNavigation())
            <ul class="-mx-2 flex flex-col gap-y-1">
            ...
```

**Reply by sinnbeck** [reply]
Ah it seesm it was changed in a later version.

```
composer update

**Reply by khanakia** [reply]
I was using v3.0.76 just upgraded and it worked. Thanks everybody for the help 😄

**Comment by iurygdeoliveira** [comment]
![Captura de tela de 2025-05-11 07-12-01](https://github.com/user-attachments/assets/4b27f742-9c2a-4abd-8e5d-02a11ebb1ac9)

I had the same problem. Notice that there is a y-gap-7 in the file indicated by @leandrocfe. I am using v3.3.14
